core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.component.spec.ts

```diff
@@ -240,6 +240,50 @@ describe('Customize Interaction Modal Component', () => {
     expect(component.isSaveInteractionButtonEnabled()).toBe(true);
   });
 
+  it('should enable Save Interaction button when no recommendation IDs' +
+  ' are provided', () => {
+    spyOn(component, 'getTitle').and
+      .returnValue(AppConstants.INTERACTION_NAMES.END_EXPLORATION);
+
+    let className = 'oppia-exploration-id-input';
+
+    spyOn(
+      document, 'getElementsByClassName'
+    ).withArgs(className).and.returnValue(undefined);
+
+    expect(component.isSaveInteractionButtonEnabled()).toBe(true);
+  });
+
+  it('should enable Save Interaction button when exploration IDs' +
+  ' are provided', () => {
+    spyOn(component, 'getTitle').and
+      .returnValue(AppConstants.INTERACTION_NAMES.END_EXPLORATION);
+
+    let inputField = document.createElement('input');
+    let inputValue = angular.element(inputField);
+    let className = 'oppia-exploration-id-input';
+
+    spyOn(
+      document, 'getElementsByClassName'
+    ).withArgs(className).and.returnValue(inputValue as any);
+
+    expect(component.isSaveInteractionButtonEnabled()).toBe(true);
+  });
+
+  it('should disable Save Interaction button when recommendation ID' +
+  ' is empty', () => {
+    spyOn(component, 'getTitle').and
+      .returnValue(AppConstants.INTERACTION_NAMES.END_EXPLORATION);
+
+    let className = 'oppia-exploration-id-input';
+
+    spyOn(
+      document, 'getElementsByClassName'
+    ).withArgs(className).and.returnValue(null);
```

Comment:
Did you confirm by running the server locally that when the input is empty, the value is null and not an empty string?

Reply:
yes the value is empty, not null in the local server, so removing the test for the NULL value